node/connection_base.js

```diff
@@ -27,6 +27,7 @@ var EventEmitter = require('./lib/event_emitter');
 
 var errors = require('./errors');
 var States = require('./reqres_states');
+var Operations = require('./operations');
```

Comment:
requests.js next to request.js and in_request.js is just confusion++

operations might not be the best word but it will be less confusing then requests